Project.toml

```
@@ -33,23 +33,23 @@ DocStringExtensions = "0.5, 0.6, 0.7, 0.8, 0.9"
 FFMPEG = "0.2, 0.3, 0.4"
 GeometryBasics = "0.3, 0.4"
 HTTP = "1"
+Meshing = "0.5, 0.6"
```

I wanted to have
```suggestion
Meshing = "0.5, 0.6, 0.7"
```
but Meshing.jl v0.7 breaks the snippets for creating a Mesh using marching cubes.

See my comment here: https://github.com/JuliaGeometry/Meshing.jl/issues/99#issuecomment-2405378939